File: 2585-delete-greatest-value-in-each-row/delete-greatest-value-in-each-row.java
class Solution {
    public int deleteGreatestValue(int[][] grid) {
       for(int[] n:grid)
       {
        Arrays.sort(n);
       }
       int sum=0;
       for(int i=0;i<grid[0].length;i++)
       {
        int max=grid[0][i];

        for(int j=0;j<grid.length;j++)
        {
            if(max<grid[j][i])
            {
                max=grid[j][i];
            }
           
        }
         sum+=max;
       }
       return sum;
    }
}

File: 2585-delete-greatest-value-in-each-row/delete-greatest-value-in-each-row.py
class Solution(object):
    def deleteGreatestValue(self, grid):
        """
        :type grid: List[List[int]]
        :rtype: int
        """
        for i in grid:
           i.sort()
        sum=0
        for i in range(len(grid[0])):
            mx=grid[0][i]
            for j in range(len(grid)):
               mx=max(mx,grid[j][i])
            sum+=mx
    
        return sum
            